README.md

```diff
@@ -21,11 +21,14 @@ Much of this project is inspired from [leafmap](https://leafmap.org/)
 * Ability to draw AOI and copy coordinates
 
 ## Contributing
-@TODO - decide and finalize on git conventions like branching standards also testing coverage
-To contribute to this codebase, clone this repository down and create a new branch. All PRs should be against the `main` branch. Branch names should be prefixed with either *feature, bugfix, docs, refactor, cleanup* and be in kebab-case format.
+To contribute to this codebase, clone this repository down and create a new branch. All PRs should be against the `main` branch. Branch names should be prefixed with either *feature, bugfix, docs, refactor, cleanup, test* and be in kebab-case format.
 
 For example when adding documentation, the branch name should look something like `docs/{special-branch-name}`. Or when refactoring for code optimization, the branch name should look something like `refactor/{special-branch-name}`
 
+### Testing
+Unit testing with Pytest
+@TODO: To Be Developed further...Think about what is the point of testing? We should just test for core pieces of functionality and not stick so closely to a threshold
+
```

Comment:
📓 : Unit testing standards to be discussed, leaving this TODO here as in code documention for us to come back too